arcgis-ios-sdk-samples/Features/Change feature layer renderer/README.md

```diff
@@ -1,18 +1,40 @@
 # Change feature layer renderer
 
-This sample demonstrates how to override and reset a renderer of a feature layer.
+Change the appearance of a feature layer with a renderer.
 
```

Comment:
The title images should be under the title section, rather than `How to use the sample` section.

Reply:
If I remember correctly, we created PRs for issues found here rather than editing directly on this PR so maybe we can fix this as part of the metadata effort?

Reply:
Ohh I see. Then I'll leave notes in this thread and fix them once I've done the metadata updates. 👍